src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/OrdinaryMethodReferenceFinder.cs

```diff
@@ -34,6 +34,7 @@ MethodKind.ReducedExtension or
 
     private static ImmutableArray<ISymbol> GetOtherPartsOfPartial(IMethodSymbol symbol)
     {
+        // https://github.com/dotnet/roslyn/issues/73772: define/use a similar helper for PropertySymbolReferenceFinder+PropertyAccessorSymbolReferenceFinder?
```

Comment:
Yeah this is MethodReferenceFinder, do not need to check this.
Maybe should check this in PropertyReferenceFinder